index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Project</title>
+    <link rel="stylesheet" href="styles/app.css">
+  </head>
+  <body>
+    <div class="header">
+      <div class="header__left">
+        <h1>Popular Github</h1>
+      </div>
+      <div class="header__right">
+        <p>people</p>
+        <p>repos</p>
+      </div>
+    </div>
+
+    <main>
+      <div class="profiles">
+        <div class="profiles__item-1">
+          <div class="user">
+            <img src="https://avatars2.githubusercontent.com/u/24994028?v=3&s=460" width="100rm" height="100rm" alt="Will Pacetti">
+              <div class="user__info">
+              <p class="user__name">William Pacetti</p>
+
+              <p class="user__username">Willpac828</p>
+              </div>
+          </div>
+          <div class="projects">
+            <div class="projects__1">
```

Comment:
Numbering here.

Another thing is group your information logically. So a project name goes together with the language it is written in instead of together with the names of other repos.

This will also help keeping things inline when you float each side.